docs/setup/keadm.md

```diff
@@ -95,6 +95,48 @@ Host has mosquit+ already installed and running. Hence skipping the installation
 KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
 ```
 
+### Enable `kubectl logs` Feature
++ First , you need to make sure you can find the kubernetes `ca.crt` and `ca.key` files. If you set up your kubernetes cluster by `kubeadm` , those files will be in `/etc/kubernetes/pki/` dir.
+
++ Set `CLOUDCOREIP` env. The environment variable is set to specify the IP address of cloudcore, or a VIP if you have a highly available cluster.
+
+  ```bash
+  export CLOUDCOREIP="172.20.12.45"
+  ```
+
++ Generate the certificates used by `CloudStream` on the cloud node.
+
+  ```bash
+  wget https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/tools/certgen.sh 
+  chmod +x certgen.sh
+  ./certgen.sh stream
+  ```
+
++ Run the following command on the host on which each apiserver runs:
+  ** Note: ** You need to set the cloudcoreip variable first
+
+  ```bash
+  iptables -t nat -A OUTPUT -p tcp --dport 10350 -j DNAT --to $CLOUDCOREIP:10003
```

Comment:
10350 and 10003 is default value, need to comment which value they point.

Reply:
Done. PTAL.

Reply:
@daixiang0  Done. PTAL.